data/json/monster_factions.json

```diff
@@ -305,13 +288,14 @@
   {
     "type": "MONSTER_FACTION",
     "name": "dermatik",
-    "base_faction": "insect"
+    "base_faction": "insect",
+    "hate": "small_animal"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "spider",
     "base_faction": "insect",
-    "neutral": [ "centipede", "bee", "wasp", "zombie" ],
+    "neutral": [ "centipede", "bee", "wasp" ],
```

Comment:
Zombies have NEUTRAL toward them; that would mean they could be attacked by spiders and not respond.

Reply:
`spider` is a child of `insect`, and they are neutral to zeds by default ( excepting the ones protecting their hives ).